cosmovisor/RELEASE_NOTES.md

```diff
@@ -0,0 +1,8 @@
+# Cosmovisor v0.1.0 Release Notes
+
+This is the first tracked release of Cosmovisor. It contains the original behavior of scanning app stdin and stdout.
+Since the original design, this release contains one important feature: state backup. Since v0.1, by default, cosmovisor will make a state backup (`<app_directory>/data` directory). Backup will be skipped if `UNSAFE_SKIP_BACKUP=false` is set.
```

Comment:
```suggestion
Since the original design, this release contains one important feature: state backup. Since v0.1, by default, cosmovisor will make a state backup (`<app_directory>/data` directory). Backup will be skipped if `UNSAFE_SKIP_BACKUP=true` is set.
```